src/test/java/com/airlines/dao/impl/CrewDaoImplTest.java

```diff
@@ -0,0 +1,227 @@
+package com.airlines.dao.impl;
+
+import com.airlines.dao.CrewDao;
+import com.airlines.model.Crew;
+import com.airlines.model.CrewMember;
+import com.airlines.util.JdbcUtil;
+import com.airlines.util.TestDatasourceProvider;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+import static com.airlines.model.Citizenship.BRAZILIAN;
+import static com.airlines.model.Citizenship.ITALIAN;
+import static com.airlines.model.Position.PILOT_IN_COMMAND;
+
+public class CrewDaoImplTest {
+    private static final String SCHEMA_SQL_PATH = "src/test/resources/sql/schema.sql";
+    private static final String DATA_SQL_PATH = "src/test/resources/sql/test_data.sql";
+    private static final String CREATE_TABLES_QUERY = JdbcUtil.getSqlQueryString(SCHEMA_SQL_PATH);
+    private static final String GENERATE_DATA_QUERY = JdbcUtil.getSqlQueryString(DATA_SQL_PATH);
+
+    private static final CrewMember CREW_MEMBER = buildTestCrewMember();
+    private static final Crew CREW = buildCrewFromDataSet();
+
+    private static CrewDao crewDao;
+
+    private static DataSource dataSource;
+
+    @BeforeClass
+    public static void init() {
+        dataSource = TestDatasourceProvider.createTestDatasource();
+        crewDao = new CrewDaoImpl(dataSource);
+    }
+
+    @Before
+    public void dataBaseInit() throws SQLException {
+        createTables();
+    }
+
+    @Test
+    public void shouldThrowExceptionIfCrewMemberWithNoIdProvidedToAddMethod() {
```

Comment:
In previous version you have two tests for 2 conditions from 4.
I left comment and recommended to cover 2 more condition in order to have all conditions covered

   if (crewMember == null || crewMember.getId() == null) {
           throw new IllegalArgumentException("Valid crewMember entity should be provided");
       }
       if (crew == null || crew.getId() == null) {
           throw new IllegalArgumentException("Valid crew entity should be provided");
       }

There 4 possible variant:
crewMember = null, valid crew
crewMember not null, but Id == null, valid crew
valid crewMember, crew = null
valid crewMember, crew not null, but crew id = null

all these variants should be covered by test.
Technically, you had just added my tests from comment to your existing tests.